Client/script_results.js

```diff
@@ -0,0 +1,84 @@
+window.onload = function() {
+    generatePage();
+  };
+
+  //const KEY = YOUR KEY GOES HERE
+  //const CX = YOUR CX CODE GOES HERE
+  
+let button = document.getElementById("submitBtn");
+button.addEventListener('click', storeSearch);
+let logo = document.getElementById("logo");
+logo.addEventListener('click', toHome); 
+  
+function storeSearch(){
+    let userInput = document.getElementById("searchInput").value
+    localStorage.setItem("searchInput", userInput);
+};
+
+async function generatePage(){
+    let query = localStorage.getItem("searchInput");
+    try {
+        let data = await fetch(`http://localhost:3000/${query}`);
```

Comment:
I believe this is the line giving me problems. I cannot actually generate a list of results for Spain, Music or Football. The I'm feeling lucky button also does not generate any results for me. I do get directed to a new page, there is just nothing on it. I think the problem is with allowing the port some flexibility as my port running was 8080. The error I get is "Failed to load resource: net::ERR_CONNECTION_REFUSED". I also go on to get a reference error I think is tied to this. "ReferenceError: data is not defined
    at getDataGoogle (script_results.js:58)
    at generatePage (script_results.js:28)". The error handling is working well, though!